packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -746,16 +746,6 @@
                             data-translate="hosting_dashboard_useful_links_user_logs"
                         ></span>
                     </oui-tile-button>
-                    <oui-tile-button
-                        aria-label="{{ ::'hosting_dashboard_service_home_ftp_newtab' | translate }}"
-                        href="{{ ftpUrl }}"
-                        external
-                        on-click="$ctrl.sendTrackClick('web::hosting::go-to-ftp-access')"
-                    >
-                        <span
-                            data-translate="hosting_dashboard_service_home_ftp"
```

Comment:
**thought:** Could you please remove it from translation file then? (only for `Messages_fr_FR.json`)
see: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:

Reply:
Hello, thanks for the review. It is done.